### ISSUE TITLE
Are the the provided Tenant Views and URLs Optional?

### ISSUE THREAD

**Issue by techdragon** [hugobessa/django-shared-schema-tenants#41]
I looked through the source code and I don't see anything that uses any of the view URLs by their names. Sometimes the Tenant system is going to be abstracted heavily away from end users, in this sort of case, I don't see a lot of use for having URLs that show details about any of the Tenant model instances, even to the users that are assigned to the Tenants. In this scenario the easy way is to just not use the provided URLs, I'm just not 100% sure this won't break something I spotted yet.

TL;DR: Are the the provided Tenant Views and their URLs ( `shared_schema_tenants.urls.*` ) optional?
